src/pages/Content/index.js

```diff
@@ -27,11 +27,51 @@ function sendMessage(message) {
 // The function receives a selector and triggers the click event on it.
 function click(selector) {
   const element = document.querySelector(selector) || null;
-  if (element) element.click();
+  if (element) {
+    try {
+      element.click();
+      return true
```

Comment:
No need to return true or false, just check for the exception.

Reply:
Oh I see. Updated the code as advised.